content/pages/the-missions/cluster-workshop.md

```diff
@@ -0,0 +1,90 @@
+expanded science opportunities, especially in the area of stellar clusters and magnetic activity. By the end of 2017, K2 
+will have observed more than 15 open clusters and associations, some on more than one occasion. The resulting data have 
+enabled ground-breaking research on a number of topics, including gyrochronology, stellar rotation and activity, planet 
+occurence rates, young stellar objects, and eclipsing binary systems. Astronomers working in these areas are warmly 
+invited to gather for discussion at BU this coming January.
+
+The motivations for this workshop include:
+
+* Summarizing the science done to date with K2 in the area of cool stars and open clusters
+* Forging new collaborations for future science with K2
+* Exploring synergies with other space missions (especially TESS), wavelength regions, and science areas.
+
+
+### Agenda
+
+This will be a small workshop with time for most participants to give a short overview of their research during the first 
+two days. (There will not be any posters.) We have reserved the final morning for "hack day" activities, including data how-to's 
+and high-level science product creation overviews. We will also encourage participants to spend time comparing technical 
+approaches to K2 data, including photometry.
+
+<h4 style="font-weight: bold;">Schedule TBA</h4>
+
+### Venue
+
+The workshop will take place on the Boston University campus.
+
+The nearest airport is the Boston Logan International Airport (BOS).
+We recommend using the MBTA system, which provides free public transportation from the airport.
+
+
+### Accommodation
+
+There is no on-site accommodation available for this meeting, but we can recommend the following nearby hotels:
+
+* <a href='https://cambridge.regency.hyatt.com/en/hotel/home.html'>The Hyatt Regency Cambridge</a> is just across
+the river and walkable (at most 15 minutes) and has rooms for approximately $150-200/night + tax.
+
+* <a href='http://www.bostonhotelbuckminster.com/'> Hotel Buckminster</a> is on the BU side of the river, in Kenmore Square.
+Prices range from $110-150/night + tax.
+
+* <a href='https://www.hotelcommonwealth.com/'> Hotel Commonwealth</a> is also in Kenmore Square, but a bit more pricey.
+
+* For other options, check out hotels on the C Green Line, such as the <a href='https://www.reservations.com/hotel/holiday-inn-boston-brookline?gclid=EAIaIQobChMIo7Hy6N3D1gIVir3tCh30-AbCEAAYAiAAEgJSyfD_BwE'>
+Holiday Inn Boston-Brookline</a> (about 20 minutes' ride).
+
+### Registration
+
+Registration will be available free of charge, apart from a small fee to be paid by cash or check on site. Please indicate 
+your [non-binding] interest in attending the workshop by <a href='https://docs.google.com/forms/d/e/1FAIpQLSdZCWOJd_L10gG-IQ9QYLkKDkkDM99Hn3awA3KStSz2B0AzCQ/viewform?c=0&w=1'> signing up for further correspondence.</a> The final date to register 
+for a talk will be December 15, 2017.
```

Comment:
Should we move it slightly earlier than Dec 15?  Folks may not book travel until they have a talk confirmed.  Let's make the initial deadline be Nov 15, make a preliminary schedule at that time, and we can then always decide to extend the talk submission deadline?